src/main/resources/application.yaml

```diff
@@ -33,25 +33,39 @@ govUkHolidays:
 #If you use a database then uncomment below lines and update db properties accordingly
 spring:
   application:
-    name: Spring Boot Template
-#  datasource:
```

Comment:
cannot we clean this up? We can always to back to git history in case we need ti back.